rtic-monotonics/src/stm32.rs

```diff
@@ -294,21 +294,13 @@ macro_rules! make_timer {
             }
 
             fn clear_compare_flag() {
-                $timer.sr().modify(|r| r.set_ccif(1, false));
+                // Do nothing, because the flags are cleared below in on_interrupt
             }
 
             fn pend_interrupt() {
                 cortex_m::peripheral::NVIC::pend(pac::Interrupt::$timer);
             }
 
-            fn enable_timer() {
-                $timer.dier().modify(|r| r.set_ccie(1, true));
-            }
-
-            fn disable_timer() {
-                $timer.dier().modify(|r| r.set_ccie(1, false));
-            }
-
```

Comment:
Please leave this. It is used by `ccr(1)`. Although it isn't absolutely crucial, but it does serve its purpose. It reduces the amount of interrupts fired in idle.